interfaces/builtin/greengrass_support.go

```diff
@@ -160,6 +269,60 @@ umount /var/snap/@{SNAP_NAME}/**,
 # sandbox for now.
 /lambda/ r,
 /lambda/** ixr,
+
+# greengrass seems to ship somewhere with an embedded version of sqlite which creates these temp files
+# ideally we would set some sort of environment variable to force sqlite to change it's temp dir as described on
+# https://bugs.launchpad.net/ubuntu-ui-toolkit/+bug/1197049
+# it could also be possible that sqlite is pulled in from one of the stage-packages in the snap though...
+owner /var/tmp/etilqs_[0-9a-zA-Z]* rw,
```

Comment:
This is not in the global namespace, it's also in the post-pivot_root area.

Reply:
@jdstrand is this access okay since it's post pivot_root or do we need to work on reworking the snap to avoid this? This access seems to be baked into greengrassd itself and would be difficult to remove at this point.